src/BenchmarkDotNet/Parameters/ParameterInstances.cs

```diff
@@ -31,10 +31,26 @@ public void Dispose()
 
         public string DisplayInfo =>  Items.Any() ? "[" + string.Join(", ", Items.Select(p => $"{p.Name}={p.ToDisplayText()}")) + "]" : "";
 
-        public string ValueInfo => Items.Any() ? "[" + string.Join(", ", Items.Select(p => $"{p.Name}={p.Value?.ToString() ?? ParameterInstance.NullParameterTextRepresentation}")) + "]" : "";
+        public string ValueInfo => Items.Any() ? "[" + string.Join(", ", Items.Select(p => $"{p.Name}={GetValueInfo(p.Value)}")) + "]" : "";
 
         public string PrintInfo => printInfo ?? (printInfo = string.Join("&", Items.Select(p => $"{p.Name}={p.ToDisplayText()}")));
 
+        private static string GetValueInfo(object value)
+        {
+            if (value is not Array array)
+                return value?.ToString() ?? "null";
+
+            //why it passes array values to cmd?
```

Comment:
There are two scenarios when we need it:

- when attaching a debugger to a running process to know what we are attaching to (imagine 2/3 params work fine and we want to attach debugger to only the last one that fails)
![image](https://user-images.githubusercontent.com/6011991/196273704-ae43b31b-e3d2-4010-a232-8029de1d6ad7.png)

- when analyzing trace files produced by `EtwProfiler` and searching for the process we care about in PerfView or other tool:
![image](https://user-images.githubusercontent.com/6011991/196274051-d2c4096f-03cb-4477-b0b5-435bd0c63191.png)

```suggestion
```